.github/workflows/pytorch_gpu_tests.yml

```
@@ -0,0 +1,40 @@
+name: Pytorch Unit Tests (with GPU) 
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'ddtrace/profiling/collector/pytorch.py'
```

```suggestion
  push:
    branches:
  	  - 'main'
  	  - 'mq-working-branch**'
  pull_request:
    paths:
      - 'ddtrace/profiling/collector/pytorch.py'
```

I think this should let us run this for main and mergequeue pushes, and for pull requests modifying the pytorch.py file